README.md

```diff
@@ -1,61 +1,75 @@
-# Setup on Mac
+# g0vhk Legco website
+香港人的線上民主平台 (<http://govhk.io>)
 
-## install python
+## Prerequisite requirements
+Python 2.7, Django 1.9.7, ImageMagick 6.x, MySQL (optional)
 
-```
-brew install python
-```
-
-## install MySQL database (if you use MySQL)
+Below is the steps to install these requirements on MacOS with [Homebrew](https://brew.sh/):
 
-```
-brew install mysql
-```
-
-## install other system dependencies
-
-wand depends on imagemagick@6
-
-```
-brew install freetype
-brew install imagemagick@6
-```
+- Install Python: `brew install python`
+- Install MySQL (optional): `brew install mysql`
+- Install Freetype: `brew install freetype`
+- Install ImageMagick 6.x: `brew install imagemagick@6`
 
-in case you have imagemagick@7 installed
+Since [Wand](https://github.com/dahlia/wand/) depends on imagemagick@6, In case you have imagemagick@7 installed
 
 ```
 brew unlink imagemagick
 brew link imagemagick@6 --force
 ```
 
-## Install django requirements
+## Getting started
 
+### Install project dependencies
 ```
+git clone https://github.com/g0vhk-io/g0vhk_legco_web.git
+cd g0vhk_legco_web
 pip install -r requirements.txt
 ```
 
-# Launch Django App
-
-## Start mysql database (if you use MySQL)
-
-change the MySQL connection settings in the config file:
+### Config database 
+change the MySQL connection settings in the config file: `gov_track_hk_web/gov_track_hk_web/settings.py`
 
+Example config (MySQL)
 ```
-gov_track_hk_web/gov_track_hk_web/settings.py
+DATABASES = {
```

Comment:
I don't recommend to type password or any credential in the settings.py (Even in development)

I would stick to this
```
DATABASES = {
    #'default': {
    #    'ENGINE': 'django.db.backends.sqlite3',
    #    'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
    #},
    'default': {
        'ENGINE': 'django.db.backends.mysql',
        'OPTIONS': {
            'read_default_file': '/etc/mysql/gov_track_hk.cnf',
        },
    }
}

```

And just as contributor to change the path of read_default_file

```
[client]
host = HOST
database = govlabhk
user = USER
password = PASSWORD
default-character-set = utf8
```